src/server/routes.ts

```diff
@@ -48,6 +49,10 @@ const appRoutesDef = {
     component: Settings,
     title: "Settings",
   },
+  "/login-email-code": {
+    component: LoginEmailCode,
+    title: "Login Email Code",
+  },
 }
 
 // Allow SMS login without kratos
```

Comment:
I think this `kratosFeatureFlag` is a thing of the past that should be deleted?

Reply:
yep, i'll remove it